aiida_quantumespresso/parsers/pp.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
+""" Parser for pp.x calculations """
+from __future__ import absolute_import
+
+import traceback
+
+import numpy as np
+
+from aiida import orm
+from aiida.common import exceptions
+
+from aiida_quantumespresso.calculations.pp import PpCalculation
+from aiida_quantumespresso.utils.mapping import get_logging_container
+from .base import Parser
+
+from six.moves import range
+from six.moves import zip
+
+class PpParser(Parser):
+    """
+    Parser for pp.x calculations
+    """
+
+    # Lookup for units
+    units_dict = {
+        0: 'e/bohr^3', # Electrons, electronic charge density
+        1: 'Ry', # Total potential
+        2: 'Ry', # Ionic potential
+        3: 'states/eV.bohr^3', # Density of states over an energy range
```

Comment:
I think this is `states/Ry.bohr^3`

From the pp.x docs:
```
   3  = local density of states at specific energy or grid of energies
        (number of states per volume, in bohr^3, per energy unit, in Ry)
```

Reply:
Beautiful